docs/spec_draft.md

```diff
@@ -343,6 +344,102 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_inference
+
+### Semantics
+
+Normalizes the `operand` tensor across batch and spatial dimensions, using
+`mean` and `variance` provided in input, for each feature in the `feature_index`
+dimension and produces a `result` tensor. More formally, the semantics can be
+expressed using MLIR syntax as follows:
+
+```mlir
+func.func @upsize(%0: tensor<2xf32>) -> tensor<2x2x2xf32> {
```

Comment:
After looking into the equivalent stablehlo code, which demonstrates how this op can be implemented using other ops, I am not sure if this helps in improving the readability of its specification. Moreover, there could be other (maybe inefficient) ways this can be implemented.

Maybe we should, 

1. include the overview of the underlying computation involved in the op  as mentioned in https://www.tensorflow.org/xla/operation_semantics#batchnormtraining. I agree that it not super simple either, but helps the reader to understand how the various inputs like mean, epsilon etc, are connected. 

2. Add comments to the implementation so as to expose its connection with (1), provided that connection  exists AND easy to point out using comments **.  

** I have not gone through the  implementation fully.  Hence not sure. Will do that later and comment further.